### PR TITLE
Sc 37/blog categories

### PR DESCRIPTION
**Pull request by casserni**

### CODE REVIEW

File: src/containers/Lists/index.tsx [hunk 1]
```diff
@@ -42,14 +43,24 @@ export const ListItem: React.FunctionComponent<IListItem> = ({ title, descriptio
   );
 };
 
-export const List: React.FunctionComponent<IList> = ({ color, hero, actionBar, pagination, items }) => {
+export const List: React.FunctionComponent<IList> = ({ color, hero, actionBar, pagination, items, buttons }) => {
+  let heroButtons: IHeroButton[] = [];
+
+  if (buttons && buttons.length) {
```
Comment (lottamus) [inline src/containers/Lists/index.tsx]: Is this needed? I think the buttons are nested inside the hero object already? https://github.com/stoplightio/stoplight.io/pull/215/files#diff-3eaa41f693485f2a11e311751a4ac971R6

Comment (casserni) [reply src/containers/Lists/index.tsx]: fixed

Comment (lottamus) [reply src/containers/Lists/index.tsx]: https://github.com/stoplightio/stoplight.io/pull/215/files#diff-61dfee42345d93dc333a369a96852402R22

Comment (lottamus) [reply src/containers/Lists/index.tsx]: https://github.com/stoplightio/stoplight.io/pull/215/files/09ec8ec6d9fad3284462005f2f971850d4549df2#diff-61dfee42345d93dc333a369a96852402R46

**Pull request merged**